src/main/java/ai/deepcode/javaclient/DeepCodeRestApi.java

```diff
@@ -35,13 +35,15 @@ private DeepCodeRestApi() {}
 
   // Create simple REST adapter which points the baseUrl.
   private static Retrofit buildRetrofit(String baseUrl) {
-    OkHttpClient client = new OkHttpClient.Builder()
+    OkHttpClient client =
+        new OkHttpClient.Builder()
             .connectTimeout(100, TimeUnit.SECONDS)
             .writeTimeout(100, TimeUnit.SECONDS)
-            .readTimeout(100, TimeUnit.SECONDS).build();
+            .readTimeout(100, TimeUnit.SECONDS)
+            .build();
     return new Retrofit.Builder()
         .baseUrl(baseUrl + "publicapi/")
-            .client(client)
+        .client(client)
```

Comment:
Please abstain from mixing formatting and functional changes in one PR. How about handle it in the separate PR?
p.s.: nice guide to PRs https://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/#856ba43cbf9148a384bea7e15f2ca563

Reply:
Sorry, my fault. Did realize to late that formatter's rule changed since I last edited this file.
Thanks for the guide!